js/main.js

```diff
@@ -12,30 +12,47 @@ const $form = document.querySelector('#form-id');
 $form.addEventListener('submit', subButton);
 
 function subButton(event) {
+  const $eform = document.querySelector('.eform');
   event.preventDefault();
-  const entry = {
-    entryId: data.nextEntryId,
-    title: event.target.elements.title.value,
-    photo: event.target.elements.myUrl.value,
-    notes: event.target.elements.notes.value
-  };
-  const $ul = document.querySelector('ul');
-  $ul.appendChild(renderEntry(entry));
+  let entry = {};
+
+  if (data.editing !== null) {
+
+    for (let i = 0; i < data.entries.length; i++) {
+      // eslint-disable-next-line eqeqeq
```

Comment:
Don't add eslint rules without asking me! If you're getting an error, it's probably for a reason